Gemfile

```diff
@@ -42,6 +42,14 @@ gem 'devise'
 
 gem 'dropzonejs-rails'
 
+gem 'coffee-script-source', '1.8.0'
+
+#gem 'haml-rails'
+
+#gem 'mongoid', '~> 4.0.0'
+#Sgem 'protected_attributes'
+gem 'jquery-rails', '~> 4.4'
```

Comment:
Rails 5.1 has dropped jQuery as a dependency from the default stack. Bootstrap 6 as well.
So you probably want to use plain cutting edge ES6.